### ISSUE TITLE
Reviewing Parsed DOM for font-size violations 

### ISSUE THREAD

**Issue by pandasa123** [priyajedi/web-doctor#4]
- After DOM manipulation and traversal is possible, as well as implementing WCAG font-size into python, we can review font-size violations for a web page
